app/email_utils.py

```diff
@@ -173,6 +174,18 @@ def send_change_email(new_email, current_email, link):
     )
 
 
+def send_pwnedpasswords_email(user, amount):
+    send_email_with_rate_control(
```

Comment:
to avoid user from having this email every time they log in, I'd recommend using `send_email_at_most_times` instead.

Reply:
I think it's good to have a reminder email though, maybe we could set nb_day to something like 30 or 90 days?